documentation/APIbacklog.md

```diff
@@ -591,7 +592,7 @@ This is a live doc that captures the status of all the APIs which have been form
       </td>
       <td><ul><li>TSC Approved (2024/09/05)</li></ul></td>
       <td>
-        <a href="https://github.com/camaraproject/SessionInsights">SessionInsights</a><br>
+        <a href="https://github.com/camaraproject/SessionInsights">QualityByDesign</a><br>

```

Comment:
```suggestion
        <a href="https://github.com/camaraproject/SessionInsights">SessionInsights</a><br>
```

I think you need to merge main into this branch and carefully only choose your own updates you made from your branch.